### PR TITLE
dont run weekly jobs on forks, because gh-tokens are not available

### PR DESCRIPTION
**Pull request by staabm**
Try to fix a error which I get in GitHub Action every week

7F430463-989D-4DC9-B5FC-AD250530B8F8

### CODE REVIEW

File: .github/workflows/weekly_pull_requests.yaml [hunk 1]
```diff
@@ -7,6 +7,10 @@ on:
 
 jobs:
     weekly_pull_requests:
+
+        # dont run jobs on forks, because gh-tokens are not available 
+        if: github.event.pull_request.head.repo.full_name == github.repository
+
```
Comment (TomasVotruba) [inline .github/workflows/weekly_pull_requests.yaml:13]: This whole workflows should only run on main repository.

Can you move the `if` to first level to make sure no other jobs will be run? As `on` is?

Comment (staabm) [reply .github/workflows/weekly_pull_requests.yaml:13]: not sure what you mean. I am not aware of the place where a `if:` can be placed above this per-job level.

Comment (TomasVotruba) [reply .github/workflows/weekly_pull_requests.yaml:13]: This one:
https://github.community/t/conditional-jobs-and-workflows/16296/10

It seems it's not possible :/

**Comment by TomasVotruba** [comment]
Thank you

**Comment by TomasVotruba** [comment]
I think the daily cron should be covered too:
https://github.com/rectorphp/rector/blob/master/.github/workflows/daily_pull_requests.yaml

**Comment by staabm** [comment]
> I think the daily cron should be covered too:

will create a followup

**Comment by staabm** [comment]
here we go: https://github.com/rectorphp/rector/pull/4813

**Comment by TomasVotruba** [comment]
Thank you

**Pull request merged**